app/vmui/packages/vmui/config-overrides.js

```diff
@@ -1,6 +1,17 @@
 /* eslint-disable */
 const { override, addExternalBabelPlugin, addWebpackAlias, addWebpackPlugin } = require("customize-cra");
 const webpack = require("webpack");
+const fs = require('fs');
+const path = require('path');
+
+// This will replace the default check
+if (process.env.REACT_APP_TYPE === "logs") {
+  const fileContent = fs.readFileSync(path.resolve(__dirname, 'public/vl_index.html'), 'utf8');
+  fs.writeFileSync(path.resolve(__dirname, 'public/index.html'), fileContent);
+} else {
+  const fileContent = fs.readFileSync(path.resolve(__dirname, 'public/vm_index.html'), 'utf8');
+  fs.writeFileSync(path.resolve(__dirname, 'public/index.html'), fileContent);
+}
```

Comment:
These changes will help remove code duplication and improve readability.

```suggestion
const pathIndexHTML = (() => {
  switch (process.env.REACT_APP_TYPE) {
    case 'logs':
      return 'public/victorialogs.html';
    case 'anomaly':
      return 'public/vmanomaly.html';
    default:
      return 'public/victoriametrics.html';
  }
})();

const fileContent = fs.readFileSync(path.resolve(__dirname, pathIndexHTML), 'utf8');
fs.writeFileSync(path.resolve(__dirname, 'public/index.html'), fileContent);
```

Reply:
thanks, bro. i have modify these code and add vmanomaly.html, please check it again.